### ISSUE TITLE
Set row's height ?

### ISSUE THREAD

**Issue by ghost** [STRML/react-grid-layout#44]
is there a way to set the row's height and not depends on its parents height ?

Responsive rowHeight = {120} breakpoints = {{ lg: 1600, md: 1024, sm: 480, xs: 320 }}
              cols = {{ lg:12, md: 12, sm: 12, xs: 12}}
        div className = "row" key = {5}
            img src = "#" 
            img src = "#" 
        /div
Responsive

**Comment by STRML** [comment]
Does the `rowHeight` option not work for you?

**Comment by zero-ralph** [comment]
if the rowHeight is inside the child it doesn't work, but if its inside the parent it works.

**Comment by STRML** [comment]
Could you rephrase this? The original post is badly garbled and I'm not quite sure what you're asking.
